eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ThrownClientLoggerCheck.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.tools.checkstyle.checks;
+
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.FullIdent;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+/**
+ * To throw an exception,
+ * <ol>
+ *   <li>Must do it through a 'logger.logAndThrow', rather than by directly calling 'throw exception'</li>
+ *   <li>Always call return after called 'logger.logAndThrow'.</li>
```

Comment:
nit: wording - should be "Always call return after calling 'logger.lowAndThrow'."